src/core/public/core_system.ts

```diff
@@ -199,6 +202,7 @@ export class CoreSystem {
       const uiSettings = await this.uiSettings.start();
       const docLinks = this.docLinks.start({ injectedMetadata });
       const http = await this.http.start();
+      const workspaces = await this.workspaces.start({ http });
```

Comment:
@SuZhou-Joe Should it pass `savedObjects` instead? like `await this.workspaces.start({ savedObjects });`
I thought workspace service will use `savedObjects` to interact with data store.

Reply:
We will use the savedObjects client in server/workspaces, and this workspaces service will only send specific http request to the router server/workspaces registered.

Let me draw a diagram to better illustrate the dependency later.